packages/cli/src/UserManagement/routes/me.ts

```diff
@@ -6,7 +6,7 @@ import express = require('express');
 import { Db, ResponseHelper } from '../..';
 import { issueJWT } from '../auth/jwt';
 import { AuthenticatedRequest, N8nApp, PublicUserData } from '../Interfaces';
-import { isValidEmail, isValidPassword, sanitizeUser } from '../UserManagementHelper';
+import { isValidEmail, isValidPassword, generatePublicUserData } from '../UserManagementHelper';
```

Comment:
would prefer to keep the `sanitizeUser` since it is more concise and describes the action better

```suggestion
import { isValidEmail, isValidPassword, sanitizeUser } from '../UserManagementHelper';
```